scripts/_lang_sync_common.py

```
@@ -240,6 +242,177 @@ def print_json_result(json_stats: dict, json_touched: list) -> None:
+    - 用节点每个已有语言值（含 zh_CN）查 index，命中词条均为候选；
+    - 候选对某缺失语言给出唯一值（且非 "?"）时才补；
+    - 只补缺失，不覆盖已有；风格跟随 zh_CN（string/pattern）。
+    返回变更列表 [(key, zh, lang, val)]。
+    """
+    touched = []
+    for key, node in data.items():
+        zh, style = _node_style(node)
+        if not zh:
+            continue
+        have = node_have_langs(node, langs + ("zh_CN",))
+        missing = [l for l in langs if l not in have]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)